docs/src/main/asciidoc/_developer-guide/Migrate_Seam_2_Booking_EAR_Binaries_-_Step_by_Step.adoc

```diff
@@ -95,10 +95,10 @@ javax/faces/api/1.2
 ----
 
 Both modules have the same module name: "javax.faces.api" but one in the
-main directory is for JSF 2.0 and the one located in the 1.2 directory
-is for JSF 1.2. If there was only one module available, we could simply
+main directory is for Faces 2.0 and the one located in the 1.2 directory
```

Comment:
`JSF 2.0` should be changed to `Jakarta Server Faces 2.3` or `Jakarta Server Faces`.  SPEC API reference link with correct name is https://jakarta.ee/specifications/faces/2.3/.

The current module names from my local wildfly master are:
```
tree ./system/layers/base/javax/faces
./system/layers/base/javax/faces
└── api
    └── main
        ├── jboss-jsf-api_2.3_spec-3.0.0.SP04.jar
        └── module.xml
```

@fjuma should we delete the documentation that explains the `JSF 1.2` module as that doesn't appear to be built anymore on WildFly master?

Reply:
@scottmarlow okay, I got it :) thanks for the update. Will wait for @fjuma to reply back on this.

Reply:
I am doing this for https://github.com/jamezp/wildfly/tree/docs-test